.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -29,8 +29,8 @@ assignees: ''
 
 > What browser are you working in?
 
-> What version of the @carbon/ibm-products (or @carbon/ibm-cloud-cognitive)
-> package are you using?
+> What version of the @carbon/ibm-products (or @carbon/ibm-products) package are
```

Comment:
This one might be worth leaving in as `(@carbon/ibm-cloud-cognitive)` to help identify if someone is on an older version?

```suggestion
> What version of the @carbon/ibm-products (or @carbon/ibm-cloud-cognitive) package are
```

Reply:
If not though, this should just be:

```suggestion
> What version of the @carbon/ibm-products package are
```